client/src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './style.css';
+
+export default ({ children, className }) => (
+  <nav className="nav">
+    <div className={className}>
+      {children.map(element => element)}
```

Comment:
@Anies12 You should be able to just put children into the jsx without mapping, and it will just work:
```
<div className={className}>
  {children}

```

Reply:
you'r right, thanks :)